### DISCUSSION TITLE
稍后再看播放有问题

### DISCUSSION THREAD

**Discussion by cjf0523** [the1812/Bilibili-Evolved#2708] [功能建议 / Ideas]
点击稍后再看中的视频相当于点了播放整个稍后再看的列表，而且鼠标无法移动到视频播放页面，没法调倍速之类的，希望能快点解决，我还挺喜欢稍后再看这个功能的
![image](https://user-images.githubusercontent.com/52514046/145817730-b62c4f4c-9aa9-4093-aeec-498b0e0aa4e2.png)

**Comment by Geeyun-JY3** [comment]
bug 应该开 issue，而且重复：#2671
